OOP/InputAction.java

```diff
@@ -0,0 +1,105 @@
+import java.util.Scanner;
+
+public class InputAction extends Menu {
+	private static Train.Type type;
```

Comment:
ACTION: This should not be a part of this class as it is only a Menu type class, a menu class should not store what option was selected. Instead it should be a part of 'RailwayReservationSystem' class, in this way it makes much sense